### ISSUE TITLE
how can i get de data-target value befor an ajax call

### ISSUE THREAD

**Issue by gmisterk** [markcell/jquery-tabledit#123]
Hi all,

Befor an ajax requset update, i want to know what cell in my table (td) is updated.

Anybody can help me to know this information.

 `<table id="my-table" method="POST" class="table">
                                        <thead>
                                            <tr>
                                                <th>Id</th>
                                                <th>Firstname</th>
                                                <th>Lastname</th>
                                                <th>Email</th>
                                            </tr>
                                        </thead>
                                        <tbody>
                                            <tr>
                                                <td >1</td>
                                                <td data-target="firstname"  ="monnom">John</td>
                                                <td data-target="lastname" >Doe</td>
                                                <td data-target="email" >john@example.com</td>
                                            </tr>
         <tr>
                                                <td >1</td>
                                                <td data-target="firstname"  ="monnom">John</td>
                                                <td data-target="lastname" >Doe</td>
                                                <td data-target="email" >john@example.com</td>
                                            </tr>
                                            
                                        </tbody>
                                    </table>`

in other words, how can i get de data-target value befor an ajax call

Thank your for any help
